openmicroscopy/bioformats
# Documentation: migration to rst as the file extension

See https://github.com/openmicroscopy/bioformats/pull/2470 for the original version of this PR

The Model and OME Files documentation sets now use the standard `.rst` file extension for Sphinx files. This PR brings  the Bio-Formats documentation in line with this convention.

To test this PR:
- check the docs builds successfully incl. the documentation bundle
- check the Octave, MATLAB & Command Line tools bundles still include the history file
- check the autogen target (`gen-structure-table`, `gen-format-pages`, `gen-meta-support`) generates `.rst` files
